src/core/lib.lua

```diff
@@ -733,8 +733,6 @@ end
 function randomseed (seed)
    seed = tonumber(seed)
    if seed then
-      local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
```

Comment:
I don't want to lose all stderr output and it's too complex to selectively filter the output at the test level, so I'll reinstate the message, but only when using a different seed from the last one.